doc/tensor.md

```diff
@@ -1327,36 +1331,66 @@ x:indexFill(2,torch.LongTensor{4,2},-10)
 ```
 
 <a name="torch.Tensor.gather"/>
-### [Tensor] gather(src, dim, index) ###
+### [Tensor] gather(dim, index) ###
```

Comment:
I'm still not convinced that this is better, because you can use it both as a method on the input tensor `src` or on the output tensor (in which case `src` is a parameter). But if you think this is clearer than I'm happy to leave it. I just want to make sure that you see both use cases.